src/sentry/conf/server.py

```diff
@@ -432,7 +432,7 @@ def env(key, default='', type=None):
 CELERY_CREATE_MISSING_QUEUES = True
 CELERY_REDIRECT_STDOUTS = False
 CELERYD_HIJACK_ROOT_LOGGER = False
-CELERY_IMPORTS = (
+CELERY_IMPORTS = [
```

Comment:
it is, but you can replace the variable. So instead of appending, you can do:

```python
settings.CELERY_IMPORTS += ('new thing',)
```

Which effectively does re-assign for a new value entirely.